src/foam/core/Argument.js

```diff
@@ -318,3 +318,30 @@ foam.CLASS({
     }
   ]
 });
+
+foam.CLASS({
+  refines: 'foam.core.AbstractMethod',
+  package: 'foam.core',
+  name: 'CreateChildRefines',
```

Comment:
Can you add documentation: to explain what this does/is-for?

Reply:
Can do. Done.